src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+ * Add a default file link format to `FileLinkFormatter` when none is provided
```

Comment:
not really a new feat to me, not sure it warrants a line here :)

Reply:
Wasn't sure about it, I agree! :+1: